distribution/scenes/textures/patterns/agate.pov

```diff
@@ -2,35 +2,38 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Agate pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Agate pattern example agate.pov.
+//
```

Comment:
If they are used to create thumbnails, I guess we can put them in everywhere? Do we know this resolution is used? The html thumbnails are mostly 160 by 120 - except when they are not and then I tried to keep the height constant. Suppose some similar approach could be taken using 240 as a fixed height.